ballerina-tests/tests/http2_mutual_ssl_test.bal

```diff
@@ -97,8 +97,63 @@ http:ClientConfiguration http2MutualSslClientConf = {
 };
 
 @test:Config {}
-public function testHttp2MutualSsl() {
-    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf);
+public function testHttp2MutualSsl1() {
+    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf1);
+    http:Response|error resp = httpClient->get("/http2Service/");
```

Comment:
If we update the function signature with `returns error?` we can simple use `check` here and omit unnecessary if-else condition.
Check for other places as well.

Reply:
All the test cases can be improved with. That is a separate task to follow I guess.

Reply:
The issue is created to track this. https://github.com/ballerina-platform/ballerina-standard-library/issues/1718